ocs_ci/deployment/deployment.py

```diff
@@ -3749,10 +3749,9 @@ def deploy(self):
             config.switch_ctx(index)
             logger.info("Creating Resource DataProtectionApplication")
             run_cmd(f"oc create -f {constants.DPA_DISCOVERED_APPS_PATH}")
-        config.switch_ctx(old_ctx)
         # Only on the active hub enable managedserviceaccount-preview
         acm_version = get_acm_version()
-
+        config.switch_acm_ctx()
```

Comment:
This line should be moved above line 3753 as get_acm_version require ACM Context